flow360/component/simulation/outputs/outputs.py

```diff
@@ -201,6 +201,32 @@ def check_unique_probe_type(cls, value):
         return _check_unique_probe_type(value, "SurfaceProbeOutput")
 
 
+class TimeAverageProbeOutput(ProbeOutput):
+    """Time average probe monitor output settings."""
+
+    # pylint: disable=abstract-method
+    frequency: int = pd.Field(default=1, ge=1)
+    frequency_offset: int = pd.Field(default=0, ge=0)
+    start_step: Union[pd.NonNegativeInt, Literal[-1]] = pd.Field(
+        default=-1, description="Physical time step to start calculating averaging"
```

Comment:
Is there effective difference in solver between start_step == 0 vs start_step == -1?

Reply:
Yes, if start_step is 0 the first time step won't be written